src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+
+/**
+ * The Deadline class represents a deadline task.
+ */
+public class Deadline extends Task {
+
+    protected String by;
+    protected LocalDate byDate;
+
+    /**
+     * Constructor for Deadline.
+     * @param description Description of the task.
+     * @param by When the task has to be done by.
+     */
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+        try {
+            LocalDate d1 = LocalDate.parse("2019-12-01");
```

Comment:
Hmm I'm not really sure but this doesn't seem to belong here? Maybe it was left here by mistake?